### ISSUE TITLE
Import person title

### ISSUE THREAD

**Issue by de-code** [elifesciences/data-pipeline#59]
Depends on #99 

Import person _title_ defined by #13 (generated by #20)

**Comment by lsh-0** [comment]
just so I'm clear, this is to extract a `title` from a `person` element in the xml, [here for example](https://github.com/elifesciences/data-pipeline/blob/develop/example-data/xml-set-1/with-person-roles-00015.xml#L12), and emit it into the generated persons CSV (which is missing a `title` field)?

**Comment by de-code** [comment]
Actually you are right, it's missing from the CSV as well. I created #99 for that.

This ticket was meant for importing it into the database (the `db-manager` project)

**Comment by lsh-0** [comment]
PR: https://github.com/elifesciences/data-pipeline/pull/100

**Comment by de-code** [comment]
I've added labels now to indicate the area of the code (mainly csv-generator or db-manager). #99 was done but I believe this issue is still open (i.e. the database doesn't populate the `title` field yet).

**Comment by lsh-0** [comment]
ok, perhaps what might be best are DODs - Definitions Of Done - simple checkboxes with a criteria for closing this particular ticket. You can write them in Github markdown like: `- [ ] feature does foo`:

- [ ] feature does foo

**Comment by de-code** [comment]
What would you suggest in this case?

a) Break the tasks down further "Add to schema", "Write/update tests", "Stage _title_ field from CSV", "Apply _title_ field from staging to dimension"?
b) Or do you mean instead of separate tickets for the "convert _title_ using csv-generator" and "import using _db-manager_".
c) Or more like test cases e.g. "Can import _title_ using db-manager and see imported _title_ in the database?

**Comment by lsh-0** [comment]
new PR: https://github.com/elifesciences/data-pipeline/pull/116
